R/palaeorotate.R

```diff
@@ -353,17 +291,10 @@ palaeorotate <- function(occdf, lng = "lng", lat = "lat", age = "age",
                       mode = dl_mode)
         }
     }
-    # Generate reference object for linking
-    assign("ref_model", readRDS(paste0(files, "/", model[1], ".RDS")))
-
-    # Get available rotation ages
-    rot_age <- colnames(ref_model)[4:ncol(ref_model)]
-    rot_age <- unique(as.numeric(sub(".*_", "", rot_age)))
-
     # Calculate rotation ages for data
-    occdf$rot_age <- rot_age[sapply(seq_len(nrow(occdf)), function(i) {
-        which.min(abs(occdf[i, age] - rot_age))
-      })]
+    occdf$rot_age <- round(occdf[, age], digits = 0)
+    # Set 0 to 1
```

Comment:
doesn't work with zeros?

Reply:
Sorry 😬 can you be a bit more specific? What doesn't work with zeros? The round function?

Reply:
Ahh it looks like you might have been reviewing an earlier commit and not the most recent one?

Reply:
Just checked, and this seems to work properly for zeros.

Reply:
Yes sorry Lewis, just checked and in worked properely ^^'.